dGame/dComponents/InventoryComponent.cpp

```diff
@@ -1010,14 +1010,33 @@ void InventoryComponent::EquipItem(Item* item, const bool skipChecks)
 		item->SetBound(true);
 	}
 
-	GenerateProxies(item);
+	std::vector<Item*> proxies;
+	
+	std::vector<Item*> generatedProxies = GenerateProxies(item);
+	proxies.insert(proxies.end(), generatedProxies.begin(), generatedProxies.end());
+
+	// Clear skills in slots where the proxies will go
+	for (const auto proxy : proxies) {
+
+		RemoveItemSkills(proxy->GetLot());
```

Comment:
Why do you remove them here if they're never equipped? If this is to fix some confusion with the client I'm not a fan of this, that means we're definitely doing something wrong in our core logic, this is just a band-aid.

Reply:
Because the RemoveItemSkills method finds which slot the proxy item goes in and removes any skills which were previously equipped in that slot. This allows me to skip that same check when calling AddItemSkills on the proxy items so they don't remove each other's skills.